.gitignore

```diff
@@ -126,3 +126,4 @@ gregorio-*
 ctan
 *.pyg
 .agignore
+*.orig
```

Comment:
Where is this extension from?  I don't recognize it.

Reply:
I use a tool for merging which leaves orig files.  I'd rather not commit them, and nothing else uses that, so I added it.

Reply:
No problem then.